### PR TITLE
Add Chime Release Notes

### PR DESCRIPTION
**Pull request by peterschwarz**

### CODE REVIEW

File: generator/source/release/chime/notes.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+---
+
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.2 (Chime) is now available.  See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.2.3/) to
+get started.
+
+This page describes the new and changed features in Hyperledger Sawtooth 1.2
+(since release 1.1).  Information is organized in the follwing categories:
```
Comment (agunde406) [inline generator/source/release/chime/notes.md]: follwing -> following

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: generator/source/release/chime/notes.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+release-name: chime
+# Copyright (c) 2019 Cargill Incorporated
+# Licensed under Creative Commons Attribution 4.0 International License
+# https://creativecommons.org/licenses/by/4.0/
+---
+
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.2 (Chime) is now available.  See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.2.3/) to
```
Comment (danintel) [inline generator/source/release/chime/notes.md:16]: Hard coding release `1.2.3` to `latest` may be a problem when future point releases are made, such as `1.2.4`. Just something to think about.

**Review by danintel — APPROVED** [review]

File: generator/source/release/chime/notes.md (hunk 2 above)
Comment (peterschwarz) [reply generator/source/release/chime/notes.md:16]: I have proposed internally that we create "release label" symlinks for bumper, chime, and so on.  We'll re-address these then.

File: generator/source/release/chime/notes.md (hunk 1 above)
Comment (peterschwarz) [reply generator/source/release/chime/notes.md]: Fixed

**Review by agunde406 — APPROVED** [review]

File: generator/source/release/chime/notes.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+## New Features
+
+- Support for [Sawtooth PBFT consensus engine version
+  1.0](https://sawtooth.hyperledger.org/docs/pbft/releases/1.0.1/).
+- Sawtooth PBFT is the recommended consensus mechanism. It is designed to handle
+  small to medium network sizes with finality and byzantine fault tolerance.
+- Sawtooth PoET is designed for larger networks at the expense of slower /
+  probabilistic finality.
+- Sawtooth Raft is designed for small to medium networks but performs more
+  slowly than Sawtooth PBFT and lacks byzantine fault tolerance.
```
Comment (annechenette) [inline generator/source/release/chime/notes.md]: Please indent these three bullets under the first "Support for Sawtooth PBFT..." bullet.  (Add two spaces at the front of each line.)

File: generator/source/release/chime/notes.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+[> back to top <](#top)
+
+## Non-breaking changes
+
+- PoET has been patched to
+  - resolve a security defect where a correctly admitted validator could game
+    its wait times. Thanks to Huibo Wang, Guoxing Chen, Yinqian Zhang,
+    Zhiqiang Lin for analyzing the poet implementation and identifying the defect.
+  - support the updated attestation service API
```
Comment (annechenette) [inline generator/source/release/chime/notes.md]: Please fix the wording and capitalization thusly:

> - PoET has the following patches:
>    - Resolve a security defect ...
>       ...
>    - Support the updated ...

**Review by annechenette — CHANGES_REQUESTED** [review]
Requested two corrections to the "new" content in the Chime release notes (since I last reviewed the draft release notes).

File: generator/source/release/chime/notes.md (hunk 4 above)
Comment (peterschwarz) [reply generator/source/release/chime/notes.md]: Fixed

File: generator/source/release/chime/notes.md (hunk 3 above)
Comment (peterschwarz) [reply generator/source/release/chime/notes.md]: Fixed

File: generator/source/release/chime/notes.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  0.4](https://sawtooth.hyperledger.org/docs/sabre/releases/0.4.0/).
+
+[> back to top <](#top)
+
+## Non-breaking changes
+
+- PoET has the following patches:
+  - Resolve a security defect where a correctly admitted validator could game
+    its wait times. Thanks to Huibo Wang, Guoxing Chen, Yinqian Zhang,
+    Zhiqiang Lin for analyzing the poet implementation and identifying the defect.
```
Comment (annechenette) [inline generator/source/release/chime/notes.md]: and Zhiqiang Lin

**Review by annechenette — APPROVED** [review]
Found one small typo (sorry - I missed it in my first pass).  I'll go ahead and approve this PR anyway.

File: generator/source/release/chime/notes.md (hunk 5 above)
Comment (peterschwarz) [reply generator/source/release/chime/notes.md]: Fixed

File: generator/source/release/bumper/notes.md [hunk 6]
```diff
@@ -12,8 +12,8 @@ release-name: bumper
 
 <a href="top"></a>
 
-Hyperledger Sawtooth 1.1 (Bumper) is now available. See the [latest
-documentation](https://sawtooth.hyperledger.org/docs/core/releases/latest/) to
+Hyperledger Sawtooth 1.1 (Bumper) is now available. See the
+[documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.1.5/) to
```
Comment (vaporos) [inline generator/source/release/bumper/notes.md:16]: Shouldn't this be releases/1.1/? (w/link on the filesystem)

Comment (peterschwarz) [reply generator/source/release/bumper/notes.md:16]: See my above comment to @danintel

**Review by rberg2 — APPROVED** [review]

**Pull request merged**